src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -363,102 +311,11 @@ private void DisplayDisabledFunctions(HttpSelfHostConfiguration config)
             }
         }
 
-        /// <summary>
-        /// Code is based on:
-        /// https://msdn.microsoft.com/en-us/library/system.configuration.configurationmanager.appsettings(v=vs.110).aspx
-        /// All connection strings are set to have providerName = System.Data.SqlClient
-        /// </summary>
-        /// <param name="connectionStrings">ConnectionStringName => ConnectionStringValue map</param>
-        private void UpdateConnectionStrings(IEnumerable<ConnectionString> connectionStrings)
-        {
-            try
-            {
-                var configFile = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
-                var settings = configFile.ConnectionStrings.ConnectionStrings;
-                settings.Clear();
-                foreach (var connectionString in connectionStrings)
-                {
-                    if (settings[connectionString.Name] == null)
-                    {
-                        settings.Add(new ConnectionStringSettings(connectionString.Name, connectionString.Value, connectionString.ProviderName));
-                    }
-                    else
-                    {
-                        // No need to update providerName as we always start off with a clean .config
-                        // every time the cli is updated.
-                        settings[connectionString.Name].ConnectionString = connectionString.Value;
-                        settings[connectionString.Name].ProviderName = connectionString.ProviderName;
-                    }
-                }
-                configFile.Save(ConfigurationSaveMode.Modified);
-                ConfigurationManager.RefreshSection(configFile.ConnectionStrings.SectionInformation.Name);
-            }
-            catch (ConfigurationErrorsException)
-            {
-                ColoredConsole.Error.WriteLine(ErrorColor("Error updating ConfigurationManager.ConnectionStrings"));
-            }
-        }
-
-        private void UpdateEnvironmentVariables(IDictionary<string, string> secrets, Uri uri)
```

Comment:
Removed, but even though this wouldn't be required by the SDK, we'll likely need to bring this back for user code.